style.css

```diff
@@ -1,238 +1,248 @@
-    list-style: none;
+  width: 100%;
+  height: 56px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 16px;
+  background: #202020;
+  flex-direction: row;
+  position: fixed;
+}
+
+header .start li,
+header .middle li,
+header .end li {
+  list-style: none;
 }
 
 header .start {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    width: 169px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 169px;
 }
 
 header .start li .toggle {
-    margin: 0;
+  margin: 0;
 }
 
 header .start li #toggle_btn {
-    border-left-width: 0px;
-    border-top-width: 0px;
-    border-bottom-width: 0px;
-    border-right-width: 0px;
-    background: #202020;
-}
-
-header .start li .toggle ion-icon {
-    height: 24px;
-    width: 24px;
-    padding: 0;
-    vertical-align: middle;
-    color: #ffffff;
-    background-color: transparent;
-    outline: none;
-    background: none;
-    margin: 0;
-    border: none;
-    padding: 0;
-    line-height: 0;
-    cursor: pointer;
+  border-left-width: 0;
+  border-top-width: 0;
+  border-bottom-width: 0;
+  border-right-width: 0;
+  background: #202020;
+}
+
+.end .ion-icon {
+  width: 100%;
+  height: 100%;
+}
+
+header .start li .toggle .ion-icon {
+  height: 24px;
+  width: 24px;
+  padding: 0;
+  vertical-align: middle;
+  color: #fff;
+  background-color: transparent;
+  outline: none;
+  margin: 0;
+  border: none;
+  line-height: 0;
+  cursor: pointer;
 }
 
 header .start li .yb_logo {
-    height: 20px;
-    width: 90px;
-    padding: 18px 14px 18px 16px;
-    flex: none;
+  height: 20px;
+  width: 90px;
+  padding: 18px 14px 18px 16px;
+  flex: none;
 }
 
 header .middle {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    flex: 0 1 728px;
-    min-width: 0px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex: 0 1 728px;
+  min-width: 0;
 }
 
 header .middle li .search_bar {
-    margin: 0 0 0 40px;
-    padding: 0 4px;
-    flex: 1;
-    color: #121212;
+  margin: 0 0 0 40px;
+  padding: 0 4px;
+  flex: 1;
+  color: #121212;
 }
 
-
-
 header .end {
-    min-width: 225px;
-    align-items: center;
-    justify-content: flex-end;
-    flex: none;
-    display: flex;
-    flex-direction: row;
+  min-width: 225px;
+  align-items: center;
+  justify-content: flex-end;
+  flex: none;
+  display: flex;
+  flex-direction: row;
 }
 
 header .end li {
-    margin: 0, 8, 0, 0;
-    padding: 8px;
-    border: 0;
-    background: transparent;
-}
-
-.end .cam-button, .end .app-button, .end .not-button, .end .finger-button {
-    color: white;
-    display: inline-block;
-    outline: none;
-    background: none;
-    margin: 0;
-    border: none;
-    padding: 0;
-    width: 24px;
-    height: 24px;
-    line-height: 0;
-    cursor: pointer;
-    vertical-align: middle;
+  margin: 0 10px 0 0;
+  padding: 8px;
+  border: 0;
+  background: transparent;
+}
+
+.end .cam-button,
```

Comment:
- [ ] There is no need to create so many classes, if they are not necessary and the only changes you are going to make our group and not individual, I recommend that you use a single class for everyone.